### PR TITLE
Added Paramatch support for multiple uses in same syberia model

### PR DESCRIPTION
**Pull request by bruno615**
Paramatch now supports multiple runs in the same syberia model.  Also cleaned up the help file and made the code easier to read. @robertzk  @iveksl2 @peterhurford

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -1,21 +1,6 @@
...
-export(paramatch)
-export(parse_datetime)
-export(remove_outliers)
-export(renamer)
-export(select_rows)
-export(select_variables)
-export(sure_independence_screen)
-export(truncator)
-export(value_replacer)
+export()
```
Comment (peterhurford) [inline NAMESPACE]: What happened here? o.O

File: R/paramatch.r [hunk 2]
```diff
@@ -14,23 +14,24 @@
 #' @examples
 #' mp <- mungebits::mungeplane(data.frame(id = c(1:2), text = c("This is an example.", "Great code uses examples like this example."), stringsAsFactors = FALSE))
 #' mb <- mungebits:::mungebit(paramatch)
-#' mb$run(mp, col = 'text', top_n = 2)
+#' mb$run(mp, col = 'text', top_n_words = 2)
```
Comment (peterhurford) [inline R/paramatch.r]: I thought it was `top_n`, not `top_n_words`.

File: NAMESPACE (hunk 1 above)
Comment (bruno615) [inline NAMESPACE]: I ran document() and it made these changes.  Maybe the newer version does not require explicit function names?

File: R/paramatch.r (hunk 2 above)
Comment (bruno615) [inline R/paramatch.r]: I switched top_n and top_n_words because I think it makes more sense for top_n = numeric and top_n_words = c(words)

Comment (peterhurford) [inline R/paramatch.r]: But your example should be `top_n` then?

File: man/paramatch.Rd [hunk 3]
```diff
@@ -29,7 +30,7 @@ This conducts full word matching only.  Substring matches do not count towards t
 \examples{
 mp <- mungebits::mungeplane(data.frame(id = c(1:2), text = c("This is an example.", "Great code uses examples like this example."), stringsAsFactors = FALSE))
```
Comment (peterhurford) [inline man/paramatch.Rd]: :)

File: NAMESPACE (hunk 1 above)
Comment (bruno615) [inline NAMESPACE]: Rerunning document() fixed this.

**Comment by robertzk** [comment]
Looks good.

**Pull request merged**